src/BenchmarksApps/HttpClientBenchmarks/Clients/HttpClient/Program.cs

```diff
@@ -0,0 +1,268 @@
+
+        if (s_metrics.SuccessRequests > 0)
+        {
+            LogPercentiledMetric("http/headers", s_metrics.HeadersTimes, TicksToMs);
+            LogPercentiledMetric("http/contentstart", s_metrics.ContentStartTimes, TicksToMs);
+            LogPercentiledMetric("http/contentend", s_metrics.ContentEndTimes, TicksToMs);
+        }
+    }
+
+    private static async Task<Metrics> Get(HttpClient client)
+    {
+        var uri = new Uri(s_options.Url + "/get");
+
+        var drainArray = new byte[81920];
+        var stopwatch = Stopwatch.StartNew();
+        
+        var metrics = new Metrics();
+        bool isWarmup = true;
+
+        var durationStopwatch = Stopwatch.StartNew();
+        while (s_isRunning)
+        {
+            if (isWarmup && !s_isWarmup)
+            {
+                if (metrics.SuccessRequests == 0)
+                {
+                    throw new Exception($"No successful requests during warmup.");
+                }
+                isWarmup = false;
+                metrics.SuccessRequests = 0;
+                metrics.BadStatusRequests = 0;
+                metrics.ExceptionRequests = 0;
+                durationStopwatch.Restart();
+            }
+            
+            try
+            {
+                stopwatch.Restart();
+                using var result = await client.GetAsync(uri, HttpCompletionOption.ResponseHeadersRead);
+                var headersTime = stopwatch.ElapsedTicks;
+                if (result.IsSuccessStatusCode)
+                {
+                    var content = await result.Content.ReadAsStreamAsync();
+                    var bytesRead = await content.ReadAsync(drainArray);
+                    var contentStartTime = stopwatch.ElapsedTicks;
+                    while (bytesRead != 0)
+                    {
+                        bytesRead = await content.ReadAsync(drainArray);
+                    }
+                    var contentEndTime = stopwatch.ElapsedTicks;
+
+                    if (!isWarmup)
+                    {
+                        metrics.HeadersTimes.Add(headersTime);
+                        metrics.ContentStartTimes.Add(contentStartTime);
+                        metrics.ContentEndTimes.Add(contentEndTime);
+                    }
+                    metrics.SuccessRequests++;
+                }
+                else
+                {
+                    Log("Bad status code: " + result.StatusCode);
+                    metrics.BadStatusRequests++;
+                }
+            }
+            catch (Exception ex)
+            {
+                Log("Exception: " + ex);
+                metrics.ExceptionRequests++;
+            }
+        }
+        var elapsed = durationStopwatch.Elapsed.TotalSeconds;
+        metrics.MeanRps = (metrics.SuccessRequests +  metrics.BadStatusRequests) / elapsed;
+        return metrics;
+    }
+
+    private static void Log(string message)
+    {
+        var time = DateTime.Now.ToString("hh:mm:ss.fff");
```

Comment:
NIT: Is the usage of `Now` intentional? Doesn't it print PCT time?

Reply:
Changed to UtcNow